OpenUpSA/wazimap-ng-ui
# Change choropleth for new data format

## Description
Getting the choropleth data on the fly instead of on the first load

## Related Issue
https://github.com/OpenUpSA/wazimap-ng-ui/issues/285

## How to test it locally
* confirm that data mapper is populated correctly
* confirm that choropleth and choropleth filtering is working correctly

## Screenshots
![image](https://user-images.githubusercontent.com/53019884/113550714-35181500-95fc-11eb-8160-989817361d7b.png)
![image](https://user-images.githubusercontent.com/53019884/113550770-4f51f300-95fc-11eb-8864-3196c0f61f5c.png)


## Changelog

### Added

### Updated
* `api.js` to make an API call whenever we need to display choropleth and don't have the data locally
* `controller.js` to update the payload when a subindicator is selected
* `mapchip.js` to fix filtering with the new data structure
* `menu.js` to populate the data mapper with the new data structure instead of using Subindicators
* `choropleth.js` to display choropleth with the new data structure
* `choropleth/subindicator_calculator.js` to calculate the choropleth to display with the new data structure
* `map/popup.js` to fix showing the choropleth data on the popup 
* `profile/subindicator_filter.js` to filtering choropleth with the new data structure

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
